src/App.js

```diff
@@ -30,7 +30,7 @@ const proteins = {
 
 const getParameterByName = (name) => {
   const url = window.location.href;
-  name = name.replace(/[\[\]]/g, "\\$&");
+  name = name.replace(/[[]]/g, "\\$&");
```

Comment:
There were warnings about unnecessary escape characters that were breaking the production build.